layouts/index.html

```diff
@@ -108,33 +108,33 @@ <h1 class="font-sans font-body font-normal">
     <section class="mb-16 px-4 md:px-0">
         <div class="container mx-auto">
             <div class="lg:flex items-center justify-center my-4">
-                <a id="tableau-logo" href="https://www.tableau.com/" target="_blank" track-click>
+                <a id="logo-home-tableau" href="https://www.tableau.com/" target="_blank" track-click>
```

Comment:
What's the benefit of tracking things like these?

Reply:
Just in case really. If we start seeing high exit rates on certain it's good to have all links tracked so we can narrow down what is going. It is also good to know how many people are actually using these links in general. My personal assumption is that not a ton of people click these links, but that is all it is an assumption. I have no data to prove or disprove my assumption. I think it is best in this case to track all links so we have the data on everything, that way we can always make data driven decisions.

Interested to hear if you have a differing opinion however. I can see how this tracking can be view as overkill.